### PR TITLE
chore: update package.json lib versions

### PR DESCRIPTION
**Pull request by droshev**
update package.json lib versions

### CODE REVIEW

File: .ci-env/flags.sh-e [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+# This flag indicates if the tag should be treated as a latest one, which is going to trigger
+# - npm publish with `latest` tag, which will become new default version
+# - main version incrementation
+latest=false
```
Comment (JKMarkowski) [inline .ci-env/flags.sh-e:5]: Should there be new file named like that?

Comment (droshev) [reply .ci-env/flags.sh-e:5]: this is autogenerated

**Review by InnaAtanasova — APPROVED** [review]

File: .ci-env/flags.sh-e (hunk 1 above)
Comment (droshev) [reply .ci-env/flags.sh-e:5]: this was autogenerated

**Review by JKMarkowski — APPROVED** [review]

**Comment by nikvalor** [comment]
Project: null CORE 
 Url: null 
 This link will expire at

**Comment by nikvalor** [comment]
Project: null Platform A 
 Url: null 
 This link will expire at

**Pull request closed without merging**